core/domain/collection_services_test.py

```diff
@@ -1009,10 +1008,9 @@ def test_record_commit_message(self):
         rights_manager.publish_exploration(self.owner_id, self.EXP_ID)
 
         collection_services.update_collection(
-            self.owner_id, self.COLLECTION_ID, _get_node_change_list(
-                self.EXP_ID,
-                collection_domain.COLLECTION_NODE_PROPERTY_ACQUIRED_SKILLS,
-                ['Skill']), 'A message')
+            self.owner_id, self.COLLECTION_ID, _get_collection_change_list(
```

Comment:
Why rename from ``_get_node_change_list`` to ``_get_collection_change_list``? ``_get_node_change_list`` still seems to be here.